### PR TITLE
Feat/new inventory list

### PR DESCRIPTION
**Pull request by wswebcreation**
> **NOTE:** The main work of all the refactoring is in this PR, that's why a lot of files were impacted.
To keep the changes per file as small as possible and to be able to change the layout per feature I split a lot of code into small components.

> **NOTE:** The inventory list item detail page will be in a different PR to keep this PR as small as possible 😉 

**Commits:**
- Initial commit:
  - refactor all components into separate components
  - refactored the app header for the new layout
  - fixed some linting issues
- Update menu and second header:
  - refactor the app header
  - refactor the drawer links
  - fix header logo bug
  - fix font to be less bold
  - refactor pages to screens
- implement new list item layout
- Add new footer:
  - add footer component including texts
- fix: fix styling:
  - only show statusbar on iOS
  - fix styling of the first header

**Overview:**
screenshot 2019-01-24 at 10 12 03

**List item**
screenshot 2019-01-24 at 10 14 29

**List item remove button and cart filled**
screenshot 2019-01-24 at 10 14 51

**Footer**
screenshot 2019-01-24 at 10 15 34

**Menu**
screenshot 2019-01-24 at 10 16 01

### CODE REVIEW

File: src/js/components/AppHeader.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+var IS_ANDROID;
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.white,
+  },
+  header_container: {
+    backgroundColor: colors.white,
+    // Add the height if the statusbar for iOS to the height of the header container
+    height: 60 + (IS_IOS ? STATUS_BAR_HEIGHT : 0),
+    // for iOS the elements can be behind the statusbar, that's why there is a padding
```
Comment (rslobojan-saucelabs) [inline src/js/components/AppHeader.js]: thanks for the explanations, very helpful!

File: src/js/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    color: colors.white,
+    fontFamily: MUSEO_SANS_NORMAL,
+    fontSize: 16,
+    textAlign: 'center',
+  },
+  footer_image: {
+    marginTop: 40,
+    flex: 1,
+    alignSelf: 'stretch',
+    // This is for keeping the aspect ratio and make it repsonse
```
Comment (rslobojan-saucelabs) [inline src/js/components/Footer.js]: typos: "make it repsonse" -> "making it responsive"

File: src/js/components/InventoryListItem.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  item_container: {
+    flexDirection: 'row',
+    paddingLeft: 25,
+    paddingRight: 25,
+    paddingTop: 40,
+  },
+  item_image: {
+    flex: 1,
+    alignSelf: 'stretch',
+    // This is for keeping the aspect ratio and make it repsonse
```
Comment (rslobojan-saucelabs) [inline src/js/components/InventoryListItem.js]: typos: "make it repsonse" -> "making it responsive"

File: src/js/components/InventoryListItem.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+          buttonStyle={ styles.button_style }
+          containerStyle={ styles.button_container_style }
+          titleStyle={ styles.button_title_style }
+          onPress={ this.addToCart }
+          title={ i18n.t('inventoryItemPage.addButton') }
+          { ...testProperties(i18n.t('inventoryListPage.addButton')) }
+        />);
+    }
+    // Needed to add `accessible={false}` to the TouchableOpacity component, otherwise the components are not accessible and shown
+    // in a flat UI structure
```
Comment (rslobojan-saucelabs) [inline src/js/components/InventoryListItem.js]: Interesting, good to know

File: src/js/config/translations/en.json [hunk 5]
```diff
@@ -126,5 +127,9 @@
     "label": "Swag Labs",
     "url": "https://saucelabs.com/",
     "404Url": "https://saucelabs.com/error/404"
+  },
+  "footer": {
+    "rights": "© 2018 Sauce Labs. All Rights Reserved.",
```
Comment (rslobojan-saucelabs) [inline src/js/config/translations/en.json]: 2019 :)

Comment (wswebcreation) [reply src/js/config/translations/en.json]: hehehe
Will change it

File: src/js/components/InventoryListItem.js (hunk 3 above)
Comment (wswebcreation) [reply src/js/components/InventoryListItem.js]: Fixed

File: src/js/components/Footer.js (hunk 2 above)
Comment (wswebcreation) [reply src/js/components/Footer.js]: Fixed it

**Comment by wswebcreation** [comment]
Thanks for the review @rslobojan-saucelabs

**Pull request merged**